source/client-side-encryption/tests/legacy/fle2v2-CreateCollection.yml

```
@@ -45,7 +43,7 @@ tests:
         arguments:
           database: *database_name
           collection: "enxcol_.encryptedCollection.esc"
-      - name: assertCollectionExists
+      - name: assertCollectionNotExists
```

I think this comment is helpful for the first occurrence of `assertCollectionNotExists` in this file.

```suggestion
      # ecc collection is no longer created for QEv2
      - name: assertCollectionNotExists
```

---

Done in https://github.com/mongodb/specifications/pull/1396/commits/40317875d1cf421ced79487d75577b947abb731f